style.css

```diff
@@ -38,14 +38,72 @@
   height: 1rem;
   margin-right: 35px;
   background-size: cover;
+  cursor: pointer;
 }
 
 .navlinks {
   display: none;
 }
 
-.navlinks.active {
-  display: flex;
+@media (max-width: 840px) {
+  .navlinks.active {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    position: absolute;
+    width: 100%;
+    height: 755px;
+    top: 0;
+    right: 0;
+    justify-content: center;
+    align-content: center;
+    background: #3c3a39;
+    cursor: auto;
+  }
```

Comment:
- [ ] One of the project's requirements is that your mobile menu design should be identical to that of the template design you chose from Figma. Kindly adjust the height of the mobile menu so that it aligns with that of the template design.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/180881428-1ca724e2-73fa-4295-9521-79121b47f008.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/180881531-03c763b3-b0a0-460c-8731-29d8eff5fe82.png)